tests/queries/0_stateless/02900_date_time_check_overflow.sql

```diff
@@ -0,0 +1,50 @@
+-- I hope we will get rid of "ignore" option one day :)
```

Comment:
People may disagree if "ignore" is a good or a bad thing. It certainly fits the current ClickHouse philosophy which is to do the fastest thing when running outside defined boundaries.
--> let's omit l. 1 and 2